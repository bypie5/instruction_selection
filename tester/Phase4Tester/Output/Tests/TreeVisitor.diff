--- SelfTestCases/TreeVisitor.out	2011-04-28 14:55:21.000000000 -0700
+++ ./Output/Tests/TreeVisitor.out	2020-03-14 14:27:34.000000000 -0700
@@ -1,44 +1,17 @@
-16
-100000000
-4
-8
-12
-14
-16
-20
-24
-28
-100000000
-50000000
-333
-333
-333
-28
-24
-333
-20
-16
-333
-333
-333
-14
-12
-8
-333
-4
-100000000
-1
-1
-1
-0
-1
-4
-8
-14
-16
-20
-24
-28
-0
-0
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 92 column 10: "100000000": operand is of incorrect type
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 165 column 10: "100000000": operand is of incorrect type
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 173 column 10: "50000000": operand is of incorrect type
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 184 column 10: "100000000": operand is of incorrect type
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 455 column 14: "$t1": operand is of incorrect type
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 460 column 14: "$t2": operand is of incorrect type
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 461 column 10: "1": operand is of incorrect type
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 515 column 14: "$t0": operand is of incorrect type
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 645 column 14: "$t0": operand is of incorrect type
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 673 column 14: "$s1": operand is of incorrect type
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 711 column 10: "1": operand is of incorrect type
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 722 column 10: "1": operand is of incorrect type
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 1111 column 14: "$t0": operand is of incorrect type
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 1138 column 14: "$s0": operand is of incorrect type
+Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 1293 column 10: "333": operand is of incorrect type
 
+Processing terminated due to errors.
