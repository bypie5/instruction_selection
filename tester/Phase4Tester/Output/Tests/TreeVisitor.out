Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 92 column 10: "100000000": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 165 column 10: "100000000": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 173 column 10: "50000000": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 184 column 10: "100000000": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 455 column 14: "$t1": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 460 column 14: "$t2": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 461 column 10: "1": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 515 column 14: "$t0": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 645 column 14: "$t0": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 673 column 14: "$s1": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 711 column 10: "1": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 722 column 10: "1": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 1111 column 14: "$t0": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 1138 column 14: "$s0": operand is of incorrect type
Error in /Users/brandonyi/Documents/School/CS179E/instruction_selection/tester/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 1293 column 10: "333": operand is of incorrect type

Processing terminated due to errors.
